### ISSUE TITLE
Uncaught ReferenceError: airportSearchCache is not defined

### ISSUE THREAD

**Issue by kausar007** [johanohly/AirTrail#7]
### The bug

No airports found issue. No matter what you search for. 

Browser console has this error

Uncaught ReferenceError: airportSearchCache is not defined

### Operating System that is running AirTrail

Docker on macOs

### AirTrail Version

v0.0.3

### Reproduction steps

1. Get docker-compose and .env file
2. run docker compose up -d
3. Create user and login
4. Click on add flight. Try to search for airport

**Comment by johanohly** [comment]
Good catch! I somehow missed an import when refactoring some files, I have hotfixed it in both v0.0.3 and latest tags on docker hub. 
You should be able to just repull the image.
